android/jsi/jsi.h

```diff
@@ -984,7 +992,7 @@ class JSI_EXPORT Value {
 
   /// Copies a Symbol lvalue into a new JS value.
   Value(Runtime& runtime, const Symbol& sym) : Value(SymbolKind) {
-    new (&data_.pointer) String(runtime.cloneSymbol(sym.ptr_));
+    new (&data_.pointer) Symbol(runtime.cloneSymbol(sym.ptr_));
```

Comment:
Could this be a breaking change?

Reply:
Honestly, I'm not sure. Why do you think it is breaking? @mganandraj in case you have thoughts

Reply:
Ooh .. Looks like this was a typo in jsi header until 0.68 which got fixed in 0.69 .. The symbol was getting converted to String !

[https://github.com/facebook/react-native/commit/da48d0dc63de085f9fcc24774a063b36c86df65a](https://github.com/facebook/react-native/commit/da48d0dc63de085f9fcc24774a063b36c86df65a)